js_13/js/script.js

```diff
@@ -2,76 +2,71 @@
 
 $(function(){
 
-  if (localStorage.getItem('sectionQA')) {
-
-    var sectionQA = localStorage.getItem('sectionQA');
+  if (!localStorage.getItem('questions')) { return; }
 
+    var questions = JSON.parse(localStorage.getItem('questions'));
     var html = $('#content-template').html();
-
     var tmpl = _.template(html);
-    var content = tmpl(sectionQA);
+    var content = tmpl(questions);
     
     $('.content').append(content);
     
-    $('.btn').on('click', function(){
-      var correctAnswers = JSON.parse(localStorage.getItem('correctAnswers'));
-      var n = 3;
+    $('.btn').on('click', function(){      
+      showModal(defineResults('.answers li input'));
+
+      $('.result-modal .btn-close').on('click', function () {
+        hideModal();
+      });
+      
+    });
+    
+    function defineResults(listElement) {
       var sectionNumber = -1;
-      var correctAnswer = -1;
+      var answerIndex = -1;
       var results = {
         id: [ 'вы не отметили', 'вы не отметили', 'вы не отметили' ]
       };
-
-      $( '.answers li input' ).each(function( index ) {
+  
+      $( listElement ).each(function( liIndex ) {
         if ($(this).prop('checked')) {
-          sectionNumber = parseInt(index / n);
-          correctAnswer = index - (sectionNumber * n);
-          if (correctAnswers.answerId[sectionNumber] === correctAnswer) {
+          sectionNumber = parseInt(liIndex / questions.length);
+          answerIndex = liIndex - (sectionNumber * questions.length);
+  
+          if (questions[sectionNumber].answers[answerIndex].isCorrect) {
             results.id[sectionNumber] = 'верно';
           } else {
-            results.id[sectionNumber] = 'не верно';
+              results.id[sectionNumber] = 'не верно';
           }
+  
         }
       });
 
-      showModal(results);
-
-      $('.result-modal .btn-close').on('click', function () {
-        hideModal();
-      });
-
-    });
-
+      return results;
+    }
 
     function showModal(result_obj) {
```

Comment:
you should keep one naming style
in your case it should be camelCase `resultObj`